src/app/shared/example-viewer/example-viewer.html

```diff
@@ -19,8 +19,14 @@
     <md-tab-group>
       <!-- TODO(jelbourn): don't hard-code the html + ts + css structure -->
       <md-tab *ngFor="let extension of ['HTML', 'TS', 'CSS']" [label]="extension">
-        <pre *ngIf="exampleFileUrl(extension)" class="docs-example-source"><doc-viewer
-            [documentUrl]="exampleFileUrl(extension)"></doc-viewer></pre>
+        <div class="docs-example-source-wrapper">
+          <button md-icon-button type="button" class="docs-example-source-copy"
```

Comment:
Last thing I noticed: needs an `aria-label` on the button for something like "Copy example source to clipboard"

(`title` unfortunately doesn't work well as a label for some screen readers)